templates/index.html

```diff
@@ -38,184 +43,263 @@ <h1 dir="ltr">IATI Data Store CSV Query Builder (Alpha Version)</h1>
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>50 rows</span>
-                  </label>
-                </li>
-                <li>
-                  <label>
-                    <input type="radio" name="entry.1352830161" value="Entire selection" id="group_1352830161_2" aria-label="Entire selection"
-                    {% if size and size == "stream=True" %}
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>Entire selection</span>
-                  </label>
-                </li>
-              </ul>
+              <div class="clearfix"></div>
+              <select data-placeholder="Choose a reporting organisation type..." multiple class="chosen-select form-control" id="entry_18398991[]" name="entry.18398991[]">
+                {{ include('include/reporting_org_type.html') }}
+              </select>
             </div>
           </div>
 
-          <div class="row form-group">
-            <label for="entry_1922375458">
-              <div>Select reporting organisation (eg UK DFID = GB-GOV-1)</div>
-              <div dir="ltr"></div>
-            </label>
+          <div class="step-section">
+            <div class="title">
+              <span class="step">2</span>
+              <h3>Select the sector</h3>
+            </div>
             <div class="clearfix"></div>
-            <select data-placeholder="Choose a reporting organisation..." multiple class="chosen-select form-control" id="entry_1922375458[]" name="entry.1922375458[]">
-              {{ include('include/reporting_org.html') }}
-            </select>
+
+            <div class="row form-group">
+              <label for="entry_1954968791">
+                <div>
+                  <a href="#" class="tooltip-ui" data-toggle="tooltip" title="Choose the sector or sectors you are looking for.">
+                    Select sector (eg Basic Health Care = 12220)
+                  </a>
+                </div>
+              </label>
+              <div class="clearfix"></div>
+              <select data-placeholder="Choose a sector..." multiple class="chosen-select form-control" id="entry_1954968791[]" name="entry.1954968791[]">
+                {{ include('include/sector.html') }}
+              </select>
+              <p class="explaination-text">For more detail of what the sectors are, see the <a href="http://iatistandard.org/202/codelists/Sector">DAC 5 Digit Sector</a> codelist.</p>
+            </div>
           </div>
 
-          <div class="row form-group">
-            <label for="entry_18398991">
-              <div>Select type of reporting organisation (eg. INGO = 21)</div>
-            </label>
+          <div class="step-section">
+            <div class="title">
+              <span class="step">3</span>
+              <h3>Select recipient geography</h3>
```

Comment:
`geography` -> `location`

(better matches the terms elsewhere in IATI, and is a 'simpler' word)